test/performance/load-test/main.go

```diff
@@ -161,9 +167,13 @@ func main() {
 	processResults(ctx, q, results)
 
 	out, err := q.Store()
-	if err != nil {
-		qclose(context.Background())
-		log.Fatalf("q.Store error: %s %v", out.String(), err)
+	alerter := alerter.Alerter{}
+	if err := alerter.SetupSlack(performance.SlackUserName, *slackReadToken, *slackWriteToken, performance.SlackChannels); err != nil {
+		log.Printf("Failed to setup slack client: %v\n", err)
+	}
+	if err := alerter.SetupGitHub("knative", "serving", *githubToken); err != nil {
+		log.Printf("Failed to setup github client: %v\n", err)
```

Comment:
We should put this in the `mako` package in `knative.dev/pkg` and read the repo information out of `config-mako`